README.md

```diff
@@ -141,3 +141,31 @@ On a Mac, it seems that it doesn't follow the default kinit functionality where
 We've built in functionality to simplify the cluster login steps.  Now within the contianer you can run `sre-login cluster-id` and it will refresh your ocm login, create a tunnel within the container if necessary, and then log-in to the cluster.
 
 `sre-login` accepts both a cluster-name or a cluster-id.  If the cluster-name is not unique, it will not ask which one, but display the clusters and exit.
+
+### Advanced scripting with ocm-container
+We've recently added the ability to run a script within the container so that you can run ocm-container within a script.
```

Comment:
not sure if we would revise this in the future, but the wording of `we've recently added` would not make sense after  a while